test/Microsoft.ML.Tests/TensorFlowEstimatorTests.cs

```diff
@@ -183,6 +183,52 @@ public void TestTensorFlow()
                 Assert.Equal(4, numRows);
             }
         }
+        
+        [TensorFlowFact]
+        public void TreatOutputAsBatched()
```

Comment:
Do we have a test for saving and loading the model?

Reply:
We do now. The `TestEstimatorCore()` method does a ton of tests, including saving and loading and making sure the schema/data are the same before/after the saving and loading.